MantidQt/CustomInterfaces/inc/MantidQtCustomInterfaces/EnggDiffraction/IEnggDiffractionView.h

```diff
@@ -343,7 +351,23 @@ class IEnggDiffractionView {
  *
  * @return bool
  */
-  virtual bool saveOutputFiles() const = 0;
+  virtual bool saveFocusedOutputFiles() const = 0;
+
+  /**
+  * Produces vanadium curves graph with three spectrum for calib
+  * output. Runsm plotSpectrum function via python.
+  *
+  */
+  virtual void plotVanCurvesCalibOutput() = 0;
+
+  /**
+  * Produces ceria peaks graph with two spectrum for calib
+  * output. Runs the plotSpectrum function via python.
```

Comment:
@FedeMPouzols Should I remove all of `Runs plotSpectrum function via python.` comment from the `IEnggDiffractionView.h` file? or just from the two methods mentioned above?